meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,130 @@
+# Wayfair plugin: Initial Setup
+
+## Prerequisites
+
+* [A Plentymarkets system](https://www.plentymarkets.co.uk).
+
+* Administrative rights on the Plentymarkets system where the Wayfair plugin will be used
+    - The Plentymarkets user's `Access` setting must be `Admin`
+    - The Plentymarkets user must be able to modify Plugin Sets
+
+* Active Wayfair supplier status
+    * A Wayfair Supplier ID is required
+    * [Information for prospective suppliers](https://partners.wayfair.com/d/onboarding/sell-on-wayfair)
+
+* [Wayfair API credentials](obtaining_credentials.md).
+
+* [Installation of the Wayfair plugin](plugin_installation.md).
+
+
+## 1. Authorizing the Wayfair Plugin to access Wayfair interfaces
+After the plugin is installed in your Plentymarkets Plugin Set, the plugin must be configured to use the correct credentials when connecting to Wayfair's interfaces.
+
+* **The authorization procedure must be performed for any Plugin Set that contains the Wayfair plugin**.
+* Copying a Plugin Set will copy the authorization information to the new plugin set.
+* An exported or imported Plugin Set may include the authorization information.
+
+The authorization steps are as follows:
+1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`
+
+2. Locate the Plugin Set that is linked to the client with which Wayfair will be used.
+
+3. Click on the `Edit` button for the desired Plugin set
+
+4. In the Wayfair row of the Plugin set, click on the `Settings` button.
+
+4. In the left-side menu, go to `Configuration` >> `Global Settings`.
+
+5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials
+
+6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md)

```

Comment:
Did something happen with the naming of the UI as it doesn't have nice labels and instead I see this:
![image](https://user-images.githubusercontent.com/22055435/87823701-e80fcb00-c841-11ea-981d-1a724bdd7dfb.png)

Reply:
See earlier comment about translations being messed up for you.